src/flask_container_scaffold/base.py

```diff
@@ -9,3 +11,14 @@
 class BaseApiModel(BaseModel):
     error: str = ''
     msg: str = ''
+
+    def __init__(self, **data):
+        warnings.simplefilter("always")
```

Comment:
(question) Is this a statement we should have in a library? If I understand [the docs](https://docs.python.org/3/library/warnings.html#overriding-the-default-filter) correctly, it will override any developer choice?

Reply:
It seems reasonable to get rid of it. Without it, we're just going to receive the first warning message.
AFAIK, the filters are an array and the first element is the filter to be used.
We can read before, this first element of the array (or just simply copy the array), to recover the value after the "warnings.warn" line.